docker-compose.yaml

```
@@ -0,0 +1,23 @@
+version: "3.9"
+
+services:
+  comfyui:
+    user: "1000:1000"
+    build: .
+    deploy:
+      resources:
+        reservations:
+          devices:
+            - driver: nvidia
+              count: 1
```

That's oddly specific. What about the dual 5090 tech bros?
```suggestion
              count: all
```

---

I have very little experience with docker-compose, I basically copied the file from the other PR this was based on. 

I don't really like that this is nvidia specific to begin with, but your suggestion sounds reasonable.